src/ray/core_worker/lib/java/org_ray_runtime_RayNativeRuntime.cc

```diff
@@ -133,6 +133,13 @@ JNIEXPORT void JNICALL Java_org_ray_runtime_RayNativeRuntime_nativeSetResource(
   THROW_EXCEPTION_AND_RETURN_IF_NOT_OK(env, status, (void)0);
 }
 
+JNIEXPORT void JNICALL Java_org_ray_runtime_RayNativeRuntime_nativeKillActor(
+    JNIEnv *env, jclass, jlong nativeCoreWorkerPointer, jbyteArray actorId) {
+  auto core_worker = reinterpret_cast<ray::CoreWorker *>(nativeCoreWorkerPointer);
+  auto status = core_worker->KillActor(JavaByteArrayToId<ActorID>(env, actorId));
+  THROW_EXCEPTION_AND_RETURN_IF_NOT_OK(env, status, (void)0);
```

Comment:
```suggestion
  THROW_EXCEPTION_AND_RETURN_IF_NOT_OK(env, status, (void) 0);
```

Reply:
clang-format suggest the the whitespace should be removed.